src/index.js

```diff
@@ -7,7 +7,7 @@ const props = {
   baseURL: 'https://spruce.dlib.indiana.edu',
   masterFileID: 'd791sg30j',
   initStructure: '',
-  mp3URL: ''
+  mp3URL: 'https://video-dev.github.io/streams/x36xhzz/x36xhzz.m3u8'
```

Comment:
Is the streaming URL always going to be of type MP3?   Or should we make this more generic, like `audioStreamUrl` or something so it doesn't suggest it needs to be an MP3?

Reply:
Oh yeah, my bad! Will fix it. And I'm trying to create a sample media object for the stand-alone application. I will put the values for that in here as well.